draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2961,6 +2963,16 @@ This approach increases the Hamming distance between inputs to different
 invocations of H, which reduces the likelihood that nonidealities in H
 affect the distribution of the b\_i values.
 
+## Target Security Levels {#security-considerations-targets}
```

Comment:
I think this paragraph should appear earlier at Section 5.1 where it is introduced `k` by the first time.

Reply:
It's a good idea to make this issue obvious in Section 5, but I think the right way to do it is to have a forward reference from Section 5 (or wherever k is first mentioned) to here. I'd strongly prefer to keep discussion of security issues to the Security Considerations section, because (roughly speaking) Section 5 is normative, whereas this discussion is informative, and keeping them distinct will make things clearer.

@armfazh do you have an idea of where in Section 5 would be a good place for a forward-reference?

Reply:
Yeah, I felt a forward reference to the informative bits from Section 5 was best.

Reply:
in Section 5.1

Reply:
> in Section 5.1

Hmm?

Reply:
I am ok with the forward reference as is the latest commit.